packages/katello/katello/katello.spec

```diff
@@ -209,17 +207,10 @@ Provides a federation of katello services
 Summary: Katello Service utilities
 Group: Applications/System
 
-# service-wait dependency
-Requires:       curl
-Requires:       ruby
-Requires:       /usr/sbin/ss
-Requires:       /bin/systemctl
```

Comment:
The `katello-service` still calls out to `systemctl` so I think this is appropriate.

Reply:
(done)